shared/src/Datadog.Trace.ClrProfiler.Native/instrumented_assembly_generator/instrumented_assembly_generator_helper.h

```diff
@@ -119,7 +119,9 @@ inline bool IsInstrumentedAssemblyGeneratorEnabled()
     {
         const auto isInstrumentedAssemblyGeneratorEnabled =
             shared::GetEnvironmentValue(cfg_instrumentation_verification_env);
-        if (isInstrumentedAssemblyGeneratorEnabled == WStr("1") ||
+        // default is true
```

Comment:
Do we have any tests that verify that we don't crash if the customer puts in a `DD_TRACE_LOG_DIRECTORY` that points to a directory that doesn't exist or we don't have write permissions to? Might be a good time to write them if not, just in case.